src/Umbraco.Core/Models/TagCacheStorageType.cs

```diff
@@ -2,7 +2,7 @@
 {
     public enum TagCacheStorageType
     {
-        Csv,
-        Json
+        Json,
```

Comment:
The order of these shouldn't change, that is sort of considered a breaking API change.

Reply:
This has now been reset to original